CHANGES.md

```diff
@@ -73,6 +73,9 @@ Released on April 19th, 2022.
     [[#1866], [#1891]]
  -  (Libplanet.Net) `IMessageCodec.Encode()` now requires a *matching*
     `PrivateKey` to be provided for `Peer`.  [[#1904], [#1905]]
+ -  `TxPolicyViolationException` and `InvalidTxException` now have a new
+    constructor that takes an `Exception` as their `Exception.InnerException`.
+    [[#1830], [#1912], [#1916]]
```

Comment:
Apparently these reference links are not [defined](https://spec.commonmark.org/0.30/#link-reference-definition).  Usually we place definitions below the section:

https://github.com/colibrishin/libplanet/blame/cb6b0b3091bbbc456e5b1f9f96dea2597931fc2c/CHANGES.md#L87-L92